app/views/users/show.html.erb

```diff
@@ -1,8 +1,90 @@
-<h3><%= @user.name %>'s Dashboard</h3>
-
-<%= button_to "Discover Movies", user_discover_path(@user), method: :get %>
-
+<div style="text-align: center;">
+  <h2><%= @user.name %>'s Dashboard</h2>
+  <%= button_to "Discover Movies", user_discover_path(@user), method: :get %></div>
 <div id="my_parties">
-<h5>My Parties</h5>
-
-</div>
\ No newline at end of file
+  <h3>My Parties</h3>
+</div>
+<div id='invited-parties'>
+  <h3>Parties I'm Hosting</h3>
+  <% @hosted_parties.each do |party| %>
```

Comment:
You'll need to define a method `hosted_parties` in the `User` model if you want to go this route. `@user` should be the only instance variable in the view

Reply:
Added hosted_parties and invited_parties to user model and changed controller and view to match. Done.